vm/src/builtins/list.rs

```diff
@@ -612,6 +602,13 @@ impl IterNext for PyListReverseIterator {
 }
 
 pub fn init(context: &Context) {
+    context
+        .types
+        .list_type
+        .slots
+        .hash
+        .store(Some(unhashable_wrapper));
```

Comment:
Maybe there is a more elegant way to do this...?

Reply:
And I missed adding unhashable_wrapper thing for deque, so test_deque fails. Deque implementation resides in collections.rs, but there is no type initialization function similar to this `init` function in `collections.rs`. Where is a good place to put this code?